bioimageio/spec/v0_3/schema.py

```diff
@@ -84,7 +84,7 @@ class RDF(BioImageIOSchema):
 
     attachments = fields.Dict(
         fields.String,
-        fields.Union([fields.URI(), fields.List(fields.URI)]),
+        # fields.Union([fields.URI(), fields.List(fields.URI)]),  # do not validate attachment values; todo: clean up
```

Comment:
```suggestion
        fields.List(fields.Union(fields.URI, fields.Dict())),
```

Reply:
A minor suggestion, or we can do without any validation.

Reply:
good idea! 👍

Reply:
Oops, we need to fill in something in `fields.list(..)`, the CI fails, sorry.